tests/subsys/power/power_mgmt_soc/prj.conf

```diff
@@ -0,0 +1,22 @@
+CONFIG_LOG=y
+
```

Comment:
We don't normally have blank lines between config settings.

Reply:
Done.